portfolio/src/main/java/com/google/sps/servlets/VideoServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Random; 
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that helps avoid getting the same video twice in a row.*/
+@WebServlet("/video")
+public class VideoServlet extends HttpServlet {
+
+  public int lastVideoIndex = 4;
```

Comment:
Can you add a comment explaining what this value is for and how you got the value 4? It's generally discouraged to have random values that aren't constants.

Reply:
Done!